pallets/utxo/src/lib.rs

```diff
@@ -297,7 +341,10 @@ pub mod pallet {
 
     #[pallet::event]
     #[pallet::generate_deposit(pub(super) fn deposit_event)]
+    #[pallet::metadata(T::AccountId = "AccountId", T::AssetId = "AssetId")]
```

Comment:
Is `AssetId` needed here? At least `TokenCreated` below doesn't use it.

Reply:
Yep, the good catch. At the moment it's no need for us.

Reply:
But for the further tokens, it will.

Reply:
Removed